components/account/manage/Edit.js

```diff
@@ -2,15 +2,15 @@ import PencilIcon from "@heroicons/react/24/outline/PencilIcon";
 
 export default function Edit({ href, label = "", children }) {
   return (
-    <div className="relative w-full">
+    <div className="relative w-full  ">
```

Comment:
I have notices formatting issues in couple of places. Format your document (Shift + ALT + F) in VS Code.

Reply:
I formatted the files in new commit. Please check